humorquotes/humorquotes/spiders/python_spider.py

```diff
@@ -1,54 +1,53 @@
 import scrapy
-from scrapy.loader import ItemLoader
 from humorquotes.items import PythonPostItems
 
 
 class PythonPostSpider(scrapy.Spider):
```

Comment:
name of file and class should be meaningful, `ReditPythonPostsSpider` shows more of what is intended by code.

Reply:
Right. I will rename this.